src/test/regress/sql/upgrade_pg_dist_object_test_after.sql

```diff
@@ -0,0 +1,24 @@
+-- drop objects from previous test (uprade_basic_after.sql) for a clean test
+-- drop upgrade_basic schema and switch back to public schema
+SET search_path to public;
+DROP SCHEMA upgrade_basic CASCADE;
+
+-- as we updated citus to available version,
+--   "isn" extension
+--   "new_schema" schema
+--   "public" schema
+-- will now be marked as distributed
+-- but,
+--   "seg" extension
+-- will not be marked as distributed
+
+-- 3 items
+SELECT * from citus.pg_dist_object ORDER BY objid;
```

Comment:
these OIDs will probably change over time

Reply:
makes sense, changing it to count(*)